lib/TestPlanner.lua

```diff
@@ -49,9 +49,19 @@ end
 	as state doesn't need to be passed around between functions or explicitly
 	global.
 ]]
-function TestPlanner.createEnvironment(builder)
+function TestPlanner.createEnvironment(builder, optionalEnv)
 	local env = {}
 
+	if optionalEnv then
+		if type(optionalEnv) ~= "table" then
+			error(("Bad argument #1 to TestPlanner.createEnvironment. Expected table, got %s"):format(typeof(optionalEnv)), 2)
```

Comment:
```suggestion
			error(("Bad argument #2 to TestPlanner.createEnvironment. Expected table, got %s"):format(typeof(optionalEnv)), 2)
```

`optionalEnv` is actually argument 2!

Reply:
fixed with latest change